hyperjumptech/monika
# Responsive mobile home page

This PR resolves #249 
### How did you implement
1. hide world-dot png on small screens
2. adapt video iframe size on small screens
3. adapt flex containers to column on small screens
    - get started & star button
    - footer
4. center text besides video on small screens
5. move `index-ntmy.js` to `index.js`

### How to test
1. `cd docs`
2. `npm run dev`
3. Check mobile responsiveness on `localhost:3000`
